modules/installation-special-config-encrypt-disk.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-encrypting-disks_{context}"]
+= Encrypting disks during installation
+
+During {product-title} installation, you can enable disk encryption on all master and worker nodes.
+This feature:
+
+* Is available for IPI and bare metal {product-title} deployments
```

Comment:
s/IPI/full stack automation

Reply:
I'll double check, but I think that, between the two encryption types, both installer- and user-provisioned infrastructures are supported. I'll say that for now and change it later if it turns out to be not true.